var/spack/repos/builtin/packages/py-itk/package.py

```diff
@@ -82,9 +116,43 @@ class PyItk(PythonPackage):
             expand=False,
         )
 
+        # version 5.3.0
+        version(
+            "5.3.0-cp37m",
+            url="https://pypi.io/packages/cp37/i/itk/itk-5.3.0-cp37-cp37m-manylinux_2_28_x86_64.whl",
+            sha256="265c8b28469164a45fd9d94c211b2ed017acc7cda7a9e74bbb20b38c49c1af61",
+            expand=False,
+        )
+        version(
+            "5.3.0-cp38",
+            url="https://pypi.io/packages/cp38/i/itk/itk-5.3.0-cp38-cp38-manylinux_2_28_x86_64.whl",
+            sha256="d83dc2b0f5d673226ef6eacac012d1da6dd36c6126f2b3cffc7ed62231c29bf2",
+            expand=False,
+        )
+        version(
+            "5.3.0-cp39",
+            url="https://pypi.io/packages/cp39/i/itk/itk-5.3.0-cp39-cp39-manylinux_2_28_x86_64.whl",
+            sha256="bcc4449f2df35224cbc26472475d2afeb8a92886a81db950b2305f911bc2a38c",
+            expand=False,
+        )
+        version(
+            "5.3.0-cp310",
+            url="https://pypi.io/packages/cp310/i/itk/itk-5.3.0-cp310-cp310-manylinux_2_28_x86_64.whl",
+            sha256="272708ee5ed5d09a519b2e98ac9c130f3146630257506ea440c83501c16f9580",
+            expand=False,
+        )
+        version(
+            "5.3.0-cp311",
+            url="https://pypi.io/packages/cp311/i/itk/itk-5.3.0-cp311-cp311-manylinux_2_28_x86_64.whl",
+            sha256="ba8361a8ed1c5462e690ee893f624c0babb7a1072a15609c26790eea717e3f77",
+            expand=False,
+        )
+
     depends_on("python@3.7.0:3.7", when="@5.1.1-cp37,5.1.2-cp37", type=("build", "run"))
     depends_on("python@3.8.0:3.8", when="@5.1.1-cp38,5.1.2-cp38", type=("build", "run"))
     depends_on("python@3.9.0:3.9", when="@5.1.2-cp39", type=("build", "run"))
```

Comment:
Need to add 5.3.0 to these

Reply:
Of course; sometimes its the simple things that get missed. Good catch; as usual for @adamjstewart!